apps/system/lockscreen/js/lockscreen_inputpad.js

```diff
@@ -147,7 +147,8 @@
   };
 
   LockScreenInputpad.prototype.updatePassCodeUI =
-  function() {
+  function(keyCode) {
```

Comment:
That doesn't have to be part of this patch, but I need more context information to know what you're doing here so I know what is in scope and what is out of scope for this patch.